flight/utils/bidirectional_dictionary.py

```diff
@@ -0,0 +1,37 @@
+"""A dictionary and can be searched by key or by value."""
```

Comment:
Really don't like this approach from a usability standpoint, no need to overcomplicate it.